<?xml version="1.0" encoding="UTF-8"?>
<robot generator="Robot 4.1 (Python 3.9.0 on linux)" generated="20210822 14:06:57.679" rpa="false" schemaversion="2">
<suite id="s1" name="Tests" source="/opt/robotframework/tests">
<suite id="s1-s1" name="Validator Tests" source="/opt/robotframework/tests/validator_tests.robot">
<kw name="Start Django and open Browser" type="SETUP">
<kw name="Start Django">
<msg timestamp="20210822 14:06:57.873" level="FAIL">No keyword with name 'Start Django' found.</msg>
<status status="FAIL" starttime="20210822 14:06:57.873" endtime="20210822 14:06:57.873"/>
</kw>
<kw name="Open Browser" library="SeleniumLibrary">
<arg>${SERVER}</arg>
<arg>${BROWSER}</arg>
<doc>Opens a new browser instance to the optional ``url``.</doc>
<status status="NOT RUN" starttime="20210822 14:06:57.873" endtime="20210822 14:06:57.874"/>
</kw>
<status status="FAIL" starttime="20210822 14:06:57.869" endtime="20210822 14:06:57.874"/>
</kw>
<test id="s1-s1-t1" name="Invalid Phone Number">
<status status="FAIL" starttime="20210822 14:06:57.874" endtime="20210822 14:06:57.875">Parent suite setup failed:
No keyword with name 'Start Django' found.</status>
</test>
<test id="s1-s1-t2" name="Valid Phone Number">
<status status="FAIL" starttime="20210822 14:06:57.875" endtime="20210822 14:06:57.876">Parent suite setup failed:
No keyword with name 'Start Django' found.</status>
</test>
<kw name="Stop Django and close browser" type="TEARDOWN">
<kw name="Close Browser" library="SeleniumLibrary">
<doc>Closes the current browser.</doc>
<status status="PASS" starttime="20210822 14:06:57.877" endtime="20210822 14:06:57.877"/>
</kw>
<kw name="Stop Django">
<msg timestamp="20210822 14:06:57.881" level="FAIL">No keyword with name 'Stop Django' found.</msg>
<status status="FAIL" starttime="20210822 14:06:57.881" endtime="20210822 14:06:57.881"/>
</kw>
<status status="FAIL" starttime="20210822 14:06:57.877" endtime="20210822 14:06:57.881">No keyword with name 'Stop Django' found.</status>
</kw>
<doc>Test suite for charger request page

This test has a workflow that is created using keywords in
the imported resource file.</doc>
<status status="FAIL" starttime="20210822 14:06:57.704" endtime="20210822 14:06:57.881">Suite setup failed:
No keyword with name 'Start Django' found.

Also suite teardown failed:
No keyword with name 'Stop Django' found.</status>
</suite>
<status status="FAIL" starttime="20210822 14:06:57.681" endtime="20210822 14:06:57.883"/>
</suite>
<statistics>
<total>
<stat pass="0" fail="2" skip="0">All Tests</stat>
</total>
<tag>
</tag>
<suite>
<stat pass="0" fail="2" skip="0" id="s1" name="Tests">Tests</stat>
<stat pass="0" fail="2" skip="0" id="s1-s1" name="Validator Tests">Tests.Validator Tests</stat>
</suite>
</statistics>
<errors>
<msg timestamp="20210822 14:06:57.865" level="ERROR">Error in file '/opt/robotframework/tests/validator_tests.robot' on line 8: Importing library 'DjangoLibrary' failed: ModuleNotFoundError: No module named 'DjangoLibrary'
Traceback (most recent call last):
  None
PYTHONPATH:
  /usr/local/bin
  /usr/local/lib/python39.zip
  /usr/local/lib/python3.9
  /usr/local/lib/python3.9/lib-dynload
  /usr/local/lib/python3.9/site-packages</msg>
</errors>
</robot>
